pluginconfig/pluginconfig-311.yaml

```diff
@@ -13,6 +13,17 @@
   #  cert: <Geveva metrics certificate value. Format: Base64 in YAML, x509.Certificate struct>
 ## Geneva integration other details
 clusterVersion: v0.0
+## Openshift component logging levels
+#levels:
```

Comment:
this is not the right place for this documentation - please remove.

Reply:
alright